packages/sdks/web-js-sdk/src/enhancers/withPersistTokens/types.ts

```diff
@@ -1,4 +1,7 @@
-export type PersistTokensOptions<A extends boolean> = {
+export type SameSite = 'Strict' | 'Lax' | 'None';
+export type CookieConfig = boolean | { SameSite: SameSite };
```

Comment:
general note about this input, I think its more consistent with JS to call the attribute `sameSite` and not `SameSite` ( I'm refering to the key, not the the type name, which is upper case)

I would change to 
```suggestion
export type CookieConfig = boolean | { sameSite: SameSite };
```

also across all inputs